dynamiqs/time_array.py

```diff
@@ -193,6 +194,11 @@ def reshape(self, *shape: int) -> TimeArray:
         """Returns a new array with the same data but of a different shape."""
         pass
 
+    @abstractmethod
+    def repeat(self, n: int, axis: int) -> TimeArray:
+        """Returns a new array with the same data but repeated `n` times along `axis`."""
+        pass
+
```

Comment:
Arf, would have been nice to rebase on #406. All the changes you made to `test_time_tensor.py` are also obsolete. I can fix it after this PR is merged if you prefer to move on with this one.

Reply:
I reverted to the current `jax` branch to avoid any issue